app/views/toolbar.js

```diff
@@ -0,0 +1,76 @@
+import Mn from 'backbone.marionette';
+import { syncFiles } from '../libs/file';
+import scdl from '../libs/soundcloud';
+import PlaylistsView from './playlists';
+import NotificationView from './notification';
+
+
+const Toolbar = Mn.LayoutView.extend({
+    
+    template: require('./templates/toolbar'),
+
+    ui: {
+        importSC: '#import-sc',
+        importText: '#import-sc-text',
+        search: '#search',
+        searchText: '#search-text'
+    },
+
+    regions: {
+        playlists: '.playlists',
+        notification: '[role="notification"]'
+    },
+
+    events: {
+        'click #sync-files': 'sync',
+        'click #import-sc': 'importStream',
+        'click #search': 'search'
+    },
+
+    sync: function() {
+        syncFiles();
+    },
+
+    onRender: function() {
+        let ui = this.ui;
+        this.showChildView('playlists', new PlaylistsView());
+        ui.importSC.focusout(function() {
+            if (ui.importText.val() == '') {
+                ui.importSC
+                    .addClass('button')
+                    .removeClass('input')
+                    .removeClass('input-focused');
+            }
+        });
+        ui.search.focusout(function() {
+            if (searchText.val() == '') {
+                ui.search.removeClass('input-focused');
+            }
+        });
+        ui.importText.keyup(function(e) {
```

Comment:
Please use class' `events` for Ui interactions definitions, don't bind them manually